CommonFunctions.ml

```diff
@@ -121,6 +121,20 @@ and get_first_release_string release =
   if release = "" then ""
   else sprintf "First published in %s." (get_release_name release)
 
+and get_deprecated_info_message_string version = 
+  match version with
+  | None -> ""
+  | Some x -> sprintf "Deprecated since %s" (get_release_name x)
+
+and get_deprecated_attribute_string version =
```

Comment:
Looks good. But I wonder if this function and the next one should go in gen_csharp_binding.ml rather than CommonFunctions.ml as they're C# only.